hls4ml/utils/example_models.py

```diff
@@ -0,0 +1,51 @@
+from urllib.request import urlretrieve
+from .config import create_vivado_config
+
+def fetch_example_model(model_name):
```

Comment:
@Duchstf  We should probably have some way to tell the user what example models are available or at the very least throw an exception if they ask for an example model that is not available.

Reply:
@jmduarte I'm afraid that there is no good way to do it without introducing extra dependencies. We can maintain a list and just print it out, but it would be a pain to update it every time we add a new model. 

I think an easier way maybe to print out the repo's address when the user try to use the function. Some thing like "Go to this address to find full list of example model". And then we can just probably throw an exception if they ask for a model that's not there.